src/login/components/App/DashboardApp/Settings/Groups/Groups.js

```diff
@@ -73,7 +73,10 @@ class Groups extends Component {
               third-party services using eduID for login.
             </p>
             <RenderDataPanel />
-            <WizardPanel />
+            <WizardPanel
+              renderCreateButton={this.renderCreateButton}
```

Comment:
could we change to arrow function `renderCreateButton={this.renderCreateButton}` ->` renderCreateButton={()=>this.renderCreateButton()}`

Reply:
👍

Reply:
I see the code sending function to wizardPanel but I wonder how "renderCreateButton" is working.